clean.nf

```diff
@@ -117,6 +117,8 @@ include {download_host; check_own; concat_contamination} from './modules/get_hos
 include {minimap2_fasta; minimap2_nano; minimap2_illumina} from './modules/minimap2'
 include {bbduk} from './modules/bbmap'
 
+include {minimap2Stats; bbdukStats; writeLog} from './modules/utils'
```

Comment:
ah cool, I did not know that this syntax can be used to import multiple processes from the same module